client/src/components/SoundPanel.tsx

```diff
@@ -0,0 +1,131 @@
+import React, { useState } from 'react';
+import { ISoundCategories, ICategory, ISoundPairs } from './BottomPanel';
+import { v4 as uuidv4 } from 'uuid';
+
+// Instruments
+import cymballIcon from '../assets/instruments/cymbalIcon.svg';
+import drumIcon from '../assets/instruments/drum.svg';
+import trumpet from '../assets/instruments/trumpet.png';
+import guitarIcon from '../assets/instruments/guitarIcon.svg';
+import gongIcon from '../assets/instruments/gong.png';
+import harpIcon from '../assets/instruments/harp.png';
+// Funny
+import gotemIcon from '../assets/funny/gottem.png';
+import noiceIcon from '../assets/funny/noice.png';
+import stop_itIcon from '../assets/funny/stop_it.png';
+import ahhhIcon from '../assets/funny/ahhh.png';
+import airIcon from '../assets/funny/air.png';
+import applauseIcon from '../assets/funny/applause.png';
+import clangIcon from '../assets/funny/clang.png';
+import groanIcon from '../assets/funny/groan.png';
+import hornIcon from '../assets/funny/horn.png';
+import laughIcon from '../assets/funny/laugh.png';
+// Nature
+import beeIcon from '../assets/nature/bee.png';
+import dogIcon from '../assets/nature/dog.png';
+import flying_foxIcon from '../assets/nature/flying_fox.png';
+import lightningIcon from '../assets/nature/lightning.png';
+import natureIcon from '../assets/nature/nature.png';
+import sealionIcon from '../assets/nature/sealion.png';
+import waterIcon from '../assets/nature/water.png';
+
+const soundCategories: ISoundCategories[] = [
+	{ category: 'Instruments', isActive: true, id: uuidv4() },
```

Comment:
the id/category seems unnecessary, ids are meant to create unique objects like when we have 100 laughing emojis each should have its own id. however in this case the categories act as ids themselves, therefore use either category or id but it doesnt look like you need uuiv4()